docs/reference/commandline/node_update.md

```diff
@@ -19,10 +19,31 @@ Update a node
 Options:
       --availability string   Availability of the node (active/pause/drain)
       --help                  Print usage
+      --label-add value       Add/update node labels (key1=value1,key2=value2)
```

Comment:
@dongluochen , can you verify, I took this from the help text here: https://github.com/docker/docker/pull/24397/files